lib/drawers/publish-page.vue

```diff
@@ -220,6 +52,13 @@
       <span class="archive-help">You may archive any page that isn't published (or scheduled to be published). Archived pages will not show up in the Clay Menu unless you explicitly filter for them.</span>
       <ui-button class="archive-submit" buttonType="button" type="primary" color="red" :disabled="isScheduled || isPublished || isArchived" @click.stop="archivePage(true)">Archive</ui-button>
     </ui-collapsible>
+
+
+    <!-- restore from published -->
```

Comment:
Random question: should we hide this tab until the page is published? Or maybe disable the dropdown if the page isn't published? I have no idea what's the best

Reply:
I debated about that when building it, but just went with disabling the actual restore button when it's not published or has no changes. But hiding it completely would work as well because I guess, why show it at all if the user can't do anything with it.

Reply:
Honestly this isn't anything I feel strongly about so I defer to your judgement!

Reply:
I went ahead and hid it when page isn't published or there are no changes.  It matches the unpublish button behavior this way, since that button is also not there until published.